src/igvfd/schemas/publication.json

```diff
@@ -112,5 +112,12 @@
             }
         }
     },
+    "fuzzy_searchable_fields": [
+        "@type",
+        "abstract",
+        "identifiers",
+        "journal",
```

Comment:
Are you trying to have "Cell" bring up all publications from Cell?

Reply:
Yeah that was my thought.